deep_object_detection/Object[] objects # Detected objects

string observation_path # The observation path 
